cscareerhub/megabot
# [bug] Fix Windows linting errors

### Purpose of pull request

**Related Issue**: resolves #162

Note: This depends on another PR #161, so please **only review the latest commit/diff for this one.**

Attempting to run linting via `yarn format` and `yarn lint` runs into two issues on Windows:
1. ESLint says that it can't find any `.js` files
2. The pre-commit hook doesn't run as Powershell and Cmd don't support the `&&` operator

Changes:
- Change ESLint command to use escaped double quotes rather than single quotes to fix this issue on Windows
- Separate the single command onto two lines so it succeeds on Windows and Mac/Linux
  - Note: The behavior is slightly different since before `yarn format` failing won't run `yarn lint`, but this is a quick and dirty fix that makes it compatible across both OS's

### Pull request checklist


### Testing instructions
Can run on both Windows and Mac/Linux

1. Attempt to lint with `yarn lint`
2. Attempt to commit changes and ensure pre-commit hook goes through properly
